JS/Dom.js

```diff
@@ -0,0 +1,20 @@
+let domUpdates = {
+
+  displayNames() {
+    $('#js-player1-name').text($('.js-player1-name-input').val())
+    $('#js-player2-name').text($('.js-player2-name-input').val())
+    $('#js-player3-name').text($('.js-player3-name-input').val())
+  },
+  // displayScore() {
+  //   $('#js-player1-score')
+  //   $('#js-player2-score')
+  //   $('#js-player3-score')
+  // }
+
+  displayCat(cats) {
+    $('#one').text(cats[0]);
+    $('#two').text(cats[1]);
+    $('#three').text(cats[2]);
+    $('#four').text(cats[3]);
```

Comment:
Numbers for ids are usually not informative enough. What are you counting? Additionally, if you need to number your ids, it's probably a sign you should be using a class instead -- it seems like you have 4 elements that represent a category.  We could select the category elements by a class instead, gain access to all 4 of them, loop over them and assign their text values with their corresponding category name.